tests/core/test_e2e.py

```diff
@@ -15,6 +15,7 @@ async def main(*, input, source_storage, assistant):
             input=input,
             source_storage=source_storage,
             assistant=assistant,
+            corpus_name="test-corpus",
```

Comment:
Unless we actually do something with `corpus_name`, should we just leave it as `None` here?